From b9e429c68d9ddd25f58fa367039e99a293fcdee6 Mon Sep 17 00:00:00 2001
From: Felix Fietkau <nbd@nbd.name>
Date: Thu, 16 Mar 2023 13:17:52 +0530
Subject: [PATCH 022/281] OpenWrt: 307-mips_highmem_offset.patch

kernel: adjust mips highmem offset to avoid the need for -mlong-calls on systems with >256M RAM

Signed-off-by: Felix Fietkau <nbd@nbd.name>

Change-Id: If19e089d0d18b20e32b420d17c832d6c514f5528
Signed-off-by: Ram Chandra Jangir <quic_rjangir@quicinc.com>
---
 arch/mips/include/asm/mach-generic/spaces.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/include/asm/mach-generic/spaces.h b/arch/mips/include/asm/mach-generic/spaces.h
index b247575c5e69..c123a003da38 100644
--- a/arch/mips/include/asm/mach-generic/spaces.h
+++ b/arch/mips/include/asm/mach-generic/spaces.h
@@ -46,7 +46,7 @@
  * Memory above this physical address will be considered highmem.
  */
 #ifndef HIGHMEM_START
-#define HIGHMEM_START		_AC(0x20000000, UL)
+#define HIGHMEM_START		_AC(0x10000000, UL)
 #endif
 
 #endif /* CONFIG_32BIT */
-- 
2.17.1

